### ISSUE TITLE
OSC não carrega como Checkout padrão.

### ISSUE THREAD

**Issue by guztavoalves** [deivisonarthur/OSC-Magento-Brasil-6-Pro#230]
Olá pessoal.
Estou com um problema na instalação do módulo.

Eu instalei o OSC 6 no Magento 1.9.0.1 e habilitei o módulo, porem no carrinho de compras quando clico para finalizar a compra, o Magento carrega o Checkout padrão e não o OSC.

Estou usando um tema personalizado, nessas condições é preciso fazer alguma alteração no XML do Checkout ou copiar os arquivos do módulo para a pasta do tema?

**Comment by rafaelstz** [comment]
verifica se na configuração em sistema > configuração está habilitado

**Comment by guztavoalves** [comment]
Está sim, o módulo está instalado e habilitado
http://prntscr.com/azjwg1

Porem ele não abre como checkout padrão do Magento.

**Comment by rafaelstz** [comment]
Copia todos os arquivos com relação ao tema em app/design/frontend/... e skin/frontend/... para a estrutura do seu tema.

**Comment by guztavoalves** [comment]
Fiz isso e continua abrindo o checkout padrão do Magento ao invés do OSC.
Fora habilitar o módulo precisa desativar algo mais  no Magento ou fazer alguma configuração de XML?

**Comment by rafaelstz** [comment]
Não, só copiar para seu tema, caso tenha sido compilado o recompile, reindexe e limpe o cache. A sua loja tem outras sublojas ou visões? Caso tenha você deverá verificar se está ativo nas outras visões.

**Comment by guztavoalves** [comment]
Então mano a loja não é compilada não e só tem a vista padrão mesmo, sem sublojas...
Eu consigo ativar o OSC 4 sem problema nenhum, mas esse, o OSC 6, não vai de jeito nenhum... muito estranho.
